app/controllers/games_controller.rb

```diff
@@ -35,6 +39,21 @@ def update
     end
   end
 
+  def forfeit
+    @game = Game.find(params[:id])
+
+    if current_user.id == @game.white_player_id
```

Comment:
Since this is triggering Rubocop warnings, can you move the logic into the Game model as a `forfeit` method?  Something like:

```ruby
def forfeit(current_user)
  winner_id = current_user.id == white_player_id ? black_player_id : white_player_id
  update_attributes(winner_id: winner_id, result: 'Forfeit')
end
```

Then your controller can just do:

```ruby
def forfeit
  @game = Game.find(params[:id])
  @game.forfeit(current_user)
  flash[:alert] = 'You have forfeited the game.'
  redirect_to games_path
end
```